internal/helm/release/manager.go

```diff
@@ -363,3 +366,59 @@ func (m manager) UninstallRelease(ctx context.Context, opts ...UninstallOption)
 	}
 	return uninstallResponse.Release, err
 }
+
+// CleanupRelease deletes resources if they are not deleted already.
+// Return true if all the resources are deleted, false otherwise.
+func (m manager) CleanupRelease(ctx context.Context, manifest string) (bool, error) {
+	dc, err := m.actionConfig.RESTClientGetter.ToDiscoveryClient()
+	if err != nil {
+		return false, fmt.Errorf("failed to get Kubernetes discovery client: %w", err)
+	}
+	apiVersions, err := action.GetVersionSet(dc)
+	if err != nil && !discovery.IsGroupDiscoveryFailedError(err) {
```

Comment:
If any other error is returned, `apiVersions` will be empty right?

Reply:
Actually I don't think `apiVersions` is used anywhere within `SortManifests()`.

Reply:
It does return the default version set but it's probably not good enough. https://github.com/helm/helm/blob/v3.4.1/pkg/action/action.go#L313 I think it's better to error out.